utils/sane-backends/patches/040-fix-saned-foreground-for-procd.patch

```diff
@@ -0,0 +1,69 @@
+Index: sane-backends-1.0.24/frontend/saned.c
```

Comment:
On 16-03-25 12:15 AM, Luiz Angelo Daros de Luca wrote:
> In utils/sane-backends/patches/040-fix-saned-foreground-for-procd.patch
> <https://github.com/openwrt/packages/pull/2543#discussion_r57418034>:
>
>> @@ -0,0 +1,69 @@
>> +Index: sane-backends-1.0.24/frontend/saned.c
>
> The package is using sane 1.0.25 while you mention 1.0.24. I worked with
> upstream project in order to reduce patches to a minimum. The next
> version will incorporate most of the "fix" patches.
>
> Please, submit this upstream. They are very welcoming. When applied

Will do.

> upstream, It would be interesting to refer it on the patch header. This

Not sure what you mean.  Where and what do you want me to put?

> helps when upstream releases a new version and I need to check every
> patch in order to check if it is needed or not.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/openwrt/packages/pull/2543/files/6faee57ddc563cd71dad384dd8c15ea48ab7184c#r57418034>
>